packages/projects/default-project/PhysicsSimulationTestSystem.ts

```diff
@@ -93,14 +105,18 @@ export const generateSimulationData = (numOfObjectsToGenerate) => {
   // TODO: Generating the object as a network object here may result in an error sometimes,
   // the reason being the spawn object network action from server is received before the object has been added to the scene locally.
   // In order to test network functionality, call the generateSimulationData directly from scene loading logic in code for now.
-  generatePhysicsObject(config, new Vector3(0, 15, 0))
+  generatePhysicsObject(config, new Vector3(0, 15, 0), true)
 }
 
 const defaultSpawnPosition = new Vector3()
+const defaultScalePosition = new Vector3(1, 1, 1)
```

Comment:
should be named `defaultScale`

Reply:
lol, copy-paste really should be avoided 😄